callisto/delivery/report_delivery.py

```diff
@@ -244,11 +247,24 @@ def render_question(self, question):
                     for q in form:
                         self.render_question(q)
                     self.pdf_elements.append(Indenter(left=-12))
-            else:
-                self.add_question(prompt)
-                self.pdf_elements.append(Indenter(left=12))
-                self.add_question('<i>None added</i>')
-                self.pdf_elements.append(Indenter(left=-12))
+
+        def _render_default():
```

Comment:
this logic isn't quite right--what you've put in render_default should be part of render_formset, it's what gets rendered when there are no forms in the formset